src/ocamlorg_frontend/pages/tutorial.eml

```diff
@@ -24,9 +24,20 @@ let of_tutorial_section (s: Data.Tutorial.Section.t) =
   | Platform -> Learn_layout.Platform
   | Guides -> Learn_layout.Guides
 
+let render_related_exercises exercises =
+  let exercise_html = exercises |> List.map (fun (exercise : Data.Exercise.t) ->
+    let link = "/exercises#" ^ exercise.number in
+    "<a href='" ^ link ^ "'>" ^ exercise.title ^ "</a>"
+  ) |> String.concat "<br>" in
+  if List.length exercises > 0 then
+    "<div class='related-exercises'><h3>Related exercises:</h3>" ^ exercise_html ^ "</div>"
+  else
+    ""
```

Comment:
While string concatenation may work (it is relatively safe to assume we will not allow XSS attacks via the exercise data), it will be better to make use of the template language to render the HTML and escape the strings we display.

In the template `platform.eml`, there's some examples of iterating over a list rendering templates within a bigger template.

Here, I would recommend to try a similar thing. Instead of concatenating strings with `br`, let's render a `ul` wrapping `li` elements that contain the links.